eclipse-scout-core/src/util/events.ts

```diff
@@ -185,6 +186,34 @@ export const events = {
     });
   },
 
+  /**
+   * @returns the app-link which triggered the given event
```

Comment:
```suggestion
   * @returns the app-link which triggered the given event, or `null` if the app-link could not be identified
```
Because the method name contains a "$", a caller might assume the result to be `$()` otherwise.